src/bundle/Resources/public/js/scripts/helpers/react.helper.js

```diff
@@ -0,0 +1,30 @@
+import { getRootDOMElement } from './context.helper';
+
+const createDynamicRoot = ({ contextDOMElement = getRootDOMElement(), id } = {}) => {
+    if (id && contextDOMElement.querySelector(`#${id}`) !== null) {
```

Comment:
`getElementById` exists only on `document` and here we have any possible element (currently - body), so I can't use it.

But at the same time, instead of contextDOMElement I can use just window.document, id has to be unique anyway...